nrgpy/quality/quality.py

```diff
@@ -149,6 +175,30 @@ def find_missing_intervals(__df, interval):
 
     return missing_timestamps, _df
 
+def find_duplicate_intervals(__df): 
```

Comment:
For new methods/functions; it would be great to use type hints like

```python
import pandas as pd

...

def find_duplicate_intervals(_df: pd.DataFrame) -> tuple[list, pd.DataFrame]:
   ...
`